package-lock.json

```diff
@@ -30499,14 +30499,15 @@
       }
     },
     "node_modules/marked": {
-      "version": "4.3.0",
-      "resolved": "https://registry.npmjs.org/marked/-/marked-4.3.0.tgz",
-      "integrity": "sha512-PRsaiG84bK+AMvxziE/lCFss8juXjNaWzVbN5tXAm4XjeaS9NAHhop+PjQxz2A9h8Q4M/xGmzP8vqNwy6JeK0A==",
+      "version": "9.0.0",
+      "resolved": "https://registry.npmjs.org/marked/-/marked-9.0.0.tgz",
+      "integrity": "sha512-37yoTpjU+TSXb9OBYY5n78z/CqXh76KiQj9xsKxEdztzU9fRLmbWO5YqKxgCVGKlNdexppnbKTkwB3RipVri8w==",
+      "dev": true,
       "bin": {
         "marked": "bin/marked.js"
       },
```

Comment:
The 'marked' package now requires a minimum Node.js version of 16. Please ensure that this requirement is met in the development, testing, and production environments.